cypress/fixtures/stubbing/DatasetManager/organizations.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { DEFAULT_ORGANIZATION } from '../default';
+
+export const ORGANIZATION_AS_USER = {
```

Comment:
I can't propose a better option but `ORGANIZATION_AS_USER` seems a bit confusing (my first thougt was "a whole organization is connecting ?") maybe we can find a name that reflects that it is default access ? `ORGANIZATION_WITH_USER_DEFAULT_ACCESS` is too long, maybe something like `ORGANIZATION_WITH_USER_ACL` ? :thinking:

Reply:
I renamed it to `ORGANIZATION_WITH_DEFAULT_ROLE_USER`, not ideal but less confusing :slightly_smiling_face: 
Thank you for the suggestion !